src/D2L.Bmx/Program.cs

```diff
@@ -282,3 +304,8 @@
 	} )
 	.Build()
 	.InvokeAsync( args );
+
+internal static partial class NativeMethods {
+	[LibraryImport( "libc", EntryPoint = "geteuid" )]
+	public static partial uint GetPosixEuid();
```

Comment:
Do you know if this works on Alpine with musl, AFAICS the musl implementation of libc, is called ``libc`` as far as imports in C etc... are concerned. 

So this might work?

Reply:
our existing libc calls work in alpine